.github/workflows/blacken.yml

```diff
@@ -0,0 +1,25 @@
+name: "Blacken"
+
+on:
+  pull_request:
+    branches:
+      - main
+
+jobs:
+  run-blacken:
+    name: blacken
+    runs-on: ubuntu-latest
+    steps:
+    - name: Checkout
+      uses: actions/checkout@v2
+    - name: Setup Python
+      uses: actions/setup-python@v2
+      with:
+        python-version: "3.8"
+    - name: Install nox
+      run: |
+        python -m pip install --upgrade setuptools pip wheel
+        python -m pip install nox
+    - name: Run blacken
+      run: |
+        nox -s blacken
```

Comment:
Does this commit back to the PR? Otherwise, `lint` in the nox session that checks if `black` has been run.

Reply:
Good catch! I've removed the blacken check.